source/Camera/Basic/CaptureHDR.m

```diff
@@ -0,0 +1,31 @@
+try
+    app = zividApplication;
+
+    disp('Connecting to camera')
+    camera = connectCamera(app);
+
+    disp('Recording HDR source images');
+    frames = NET.createGeneric('System.Collections.Generic.List',{'Zivid.NET.Frame'});
+
+    settings = camera.Settings;
+    for iris = [14 21 35]
+        disp(['Capture frame with iris = ' num2str(iris)]);
+        settings.Iris = iris;
+        camera.SetSettings(settings)
+        frames.Add(camera.Capture());
+    end
+
+    disp('Creating HDR frame');
+    hdrFrame = Zivid.NET.HDR.CombineFrames(frames);
+
+    disp('Saving the frames')
+    frame(1).Save('14.zdf');
+    frame(1).Save('21.zdf');
+    frame(1).Save('35.zdf');
+    hdrFrame.Save('HDR.zdf');
```

Comment:
I guess you mean to itterate those number? And why save all the frames?

Reply:
This is how it is for other programming language samples and it will change eventually (most likely for 2.0). We will not do separate single captures and hdr merge but instead, just one capture hdr directly, so we will end up saving only the hdrFrame.

For now, I want to keep it consistent with other languages.

Reply:
But you still want to iterate? you're saving the same frame three times??

Reply:
Done